lib/Transform/XTenMinimizeLiveTensors.cpp

```diff
@@ -148,6 +154,13 @@ SmallVector<Value> getFmOperands(Operation *op) {
   if (isInCoreChainSubgraph(op))
     return getSubgraphIFMs(op);
 
+  if (isConcatSubgraph(op))
+    return op->getOperands();
+
+  if (isa<torch::Torch::PrimListConstructOp>(op)) {
```

Comment:
I know what is happening here because you explained it to me, but it would help to draw what it was added to support (maybe a cut-down torch concat+related lists snippet and how this gets to `Torch::PrimListConstructOp` and mentioning that the `Torch::ListType` is treated as contributing zero size).

Reply:
I added some more comments.